### DISCUSSION TITLE
Changes in the media info page

### DISCUSSION THREAD

**Discussion by Shah-do** [DonutWare/Fladder#181] [Ideas]
### Summary

I think it'd be better to move the play button and its adjacent elements further up as the play button being available only after scrolling feels unintuitive. At the same time, I feel the media Title is irrelevant as the logo is already there on the info page and the title should appear only when a logo isn't available and at the same spot as the logo does otherwise. This would give the page a cleaner look overall.

### Basic Example

![Frame 5](https://github.com/user-attachments/assets/fd8c4de7-5ada-497e-bc72-c2c18e94ba7f)

### Drawbacks

The drawbacks can be that the audio and subtitles selection option might not fit so will have to be pushed down

**Comment by PartyDonut** [comment]
Hey thanks for the suggestion, I will include the ideas in this ticket #127 as they are related to the same screen.
If you have any other idea's let me know in that ticket.
